source/documentation/standards/naming-domains.html.md.erb

```diff
@@ -81,39 +85,50 @@ For example: `staff.staging.prisonvisits.service.gov.uk`.
-should also use authentication (such as HTTP basic auth) to prevent
-public users who come across them thinking they’re real.
+Non-production environments and [hosted prototypes]
+(https://www.gov.uk/service-manual/design/making-prototypes#sharing-code-prototypes)
+should also use authentication (such as HTTP basic auth) to prevent public
+users who come across them thinking they’re real.
+
+Non-production environments can also use different styling as a way of making it
+clear to users which environment they’re in - but that’s more useful for people
+working on the service to not modify data in production than to keep users out.
+GOV.UK publishing apps do this, for example compare GOV.UK Signon in their
+[Staging](https://signon.staging.publishing.service.gov.uk) and [Production]
+(https://signon.publishing.service.gov.uk) environments.
+
+## Non `.gov.uk` domains
 
-Pre-production environments can also use different styling as a way of
-making it clear to users which environment they’re in - but that’s more
-useful for people working on the service to not modify data in
-production than to keep users out. GOV.UK publishing apps do this,
-for example compare GOV.UK Signon in their [Staging](https://signon.staging.publishing.service.gov.uk)
-and [Production](https://signon.publishing.service.gov.uk) environments.
+Domains that don't end in `.gov.uk` should not be used for MOJ services.
 
-## Pitfalls of non `.gov.uk` domains
+If you use one, please refer to
+[How to get, register or manage a domain name](how-to-get-a-domain-name.html).
+The MOJ Domains team will provide you with a `.gov.uk` domain as set out above,
+and will help you deprecate your old domain.
 
-You should follow the above advice for these sorts of reasons:
+You should use a `.gov.uk` subdomain, as above, for these reasons:
 
 - Public government things should on principle be on government domains
-- The ownership of a gov domain can be tracked to a government body by a member of the public
-- By using .gov.uk we never have the problem that a domain's control is lost to a previous supplier or staff member who set it up and moved on. Domain renewals are easier to keep on top of when centralized. When the domain is not renewed, it can be snapped up by a squatter.
-- The Service Manual says [not to use the crown, New Transport font etc](https://www.gov.uk/service-manual/design/making-your-service-look-like-govuk#if-your-service-isnt-on-govuk))
-- Unfamiliar domains look untrustworthy, putting off users including internal staff from using the service
+- The ownership of a gov domain can be tracked to a government body by a member
+  of the public
+- By using .gov.uk we never have the problem that a domain's control is lost to
+  a previous supplier or staff member who set it up and moved on
+- Domain renewals are easier to keep on top of when centralized. When the domain
+  is not renewed, it can be bought a squatter and used for phishing and scams
```

Comment:
should this be "bought **by** a squatter"?

Reply:
😄